src/Components/NavBar.jsx

```diff
@@ -0,0 +1,35 @@
+import Container from 'react-bootstrap/Container';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import NavDropdown from 'react-bootstrap/NavDropdown';
+
+
+const NavBar = () => {
+    return (
+        <Navbar bg="light" expand="lg" sticky='top'>
+        <Container>
+          <Navbar.Brand href="#home">Navigation Bar</Navbar.Brand>
+          <Navbar.Toggle aria-controls="basic-navbar-nav" />
+          <Navbar.Collapse id="basic-navbar-nav">
+            <Nav className="me-auto">
+              <Nav.Link href="#home">Home</Nav.Link>
+              <Nav.Link href="#articles">Articles</Nav.Link>
+              <NavDropdown title="Topics" id="basic-nav-dropdown">
+                <NavDropdown.Item href="#action/3.1">Coding</NavDropdown.Item>
```

Comment:
I know this has been added for simplicity but having all the topics be a separate page is probably the next ticket you want to work on :)